src/users/users.gateway.ts

```diff
@@ -48,61 +41,44 @@ export class UsersGateway implements OnGatewayDisconnect {
   private readonly logger = new Logger(UsersGateway.name);
   constructor(private readonly usersService: UsersService) {}
 
-  handleDisconnect(client: SocketDto) {
-    this.usersService.safeDisconnect(client);
-  }
-
   /**
    * connects notLoggedin user on the website to the app
```

Comment:
is this description still correct?

Reply:
I think this is registering and authorizing, but not connecting two clients yet

Reply:
ill fix comments